### ISSUE TITLE
[Crash] System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)

### ISSUE THREAD

**Issue by JoveToo** [sn4k3/UVtools#813]
### System

```Text
UVtools v4.0.6 X64
Operative system: Linux 6.6.6-200.fc39.x86_64 #1 SMP PREEMPT_DYNAMIC Mon Dec 11 17:29:08 UTC 2023 X64
Processor: AMD Ryzen 9 7950X3D 16-Core Processor
Processor cores: 32
Memory RAM: 17.07 / 30.55 GB
Runtime: fedora.39-x64
Framework: .NET 6.0.25
AvaloniaUI: 11.0.6
OpenCV: 4.8.1

Path:       /tmp/.mount_UVToolQbznym/usr/bin/
Executable: /tmp/.mount_UVToolQbznym/usr/bin/UVtools
```

### Printer and Slicer

- Printer:  Anycubic M5s
- Slicer:  Lychee

### Description of the bug

```
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)
 ---> Tmds.DBus.Protocol.DBusException: org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service
   at Tmds.DBus.Protocol.DBusConnection.MyValueTaskSource`1.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at Tmds.DBus.Protocol.DBusConnection.CallMethodAsync(MessageBuffer message)
   at Tmds.DBus.Protocol.Connection.CallMethodAsync(MessageBuffer message)
   --- End of inner exception stack trace ---
Category: Task
File: Pandemic The Cure pin x2.pm5s  [Version: 518] [Class: PhotonWorkshopFile]

Machine date time: 12/30/2023 2:01:56AM
    UTC date time: 12/30/2023 1:01:56AM
```

### How to reproduce

I open any .pm5s.
It crashes  immediately upon selecting the file, not when it goes back the main screen and is  importing the file.
It started with 4.0.5.

**Comment by sn4k3** [comment]
Duplicate of #803
